frida_tools/tracer.py

```diff
@@ -636,7 +636,7 @@ def ensure_handler(self, target):
             handler_dir = os.path.dirname(handler_file)
             if not os.path.isdir(handler_dir):
                 os.makedirs(handler_dir)
-            with open(handler_file, 'w') as f:
+            with open(handler_file, 'w', encoding='utf-8') as f:
```

Comment:
Should use `codecs.open()` here as we still support Python 2.7. (Will be dropped soon though but in the meantime we don't want to break it.)